amd64/src/semantic.rs

```diff
@@ -747,7 +755,7 @@ pub fn call(a: Rvalue) -> Result<(Vec<Statement>, JumpSpec)> {
         call ?, new_rip:64;
     });*/
     let stmts = rreil!{
-        call ?, (a);
```

Comment:
Ah, k.

So if possible, keep in mind a binary encoding when designing the call op; variable length arguments make encoding system significantly more complicated.

I mention this because I'm 100% convinced the quickest, fastest, and most maintainable way to make panopticon really, _really_ fast is to go the binary encoding route for the il, which will reduce a libc analysis from 2gb -> 200mb, and secondly, to implement your suggestions for function having the basic blocks in a single vector, and not allocated everywhere all over the heap